pkgs/tools/wayland/chayang/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchFromSourcehut
+, meson
+, ninja
+, pkg-config
+, wayland-scanner
+, wayland-protocols
+, wayland
+}:
+
+stdenv.mkDerivation rec {
+  pname = "chayang";
+  version = "0.1.0";
+
+  src = fetchFromSourcehut {
+    owner = "~emersion";
+    repo = pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21